fieldservice_stage_validation/models/fsm_location.py

```diff
@@ -13,6 +13,8 @@ def _validate_stage_fields(self):
             stage = rec.stage_id
```

Comment:
Maybe a bit cleaner is `for rec in self.filtered(lambda x: x.stage_id.validate_field_ids):`